backend/app/services/git/schema.py

```diff
@@ -0,0 +1,5 @@
+from pydantic import BaseModel, conlist
+
+
+class PullRequestData(BaseModel):
+    paths: conlist(str, min_items=1) = []
```

Comment:
As for one usage in code I find this unnecessary complicating typing. Why not to use default list instead?